Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,30 @@
-/* All styles for your Card Components live here */
+.contentContainer {
```

Comment:
good job on your CSS. I know it's been a hot minute since we used vanilla CSS like this so I'm glad you were able to flex this skills again without problems.